src/m365/aad/commands/app/app-add.ts

```diff
@@ -104,7 +113,16 @@ class AadAppAddCommand extends GraphCommand {
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
     this
       .resolveApis(args, logger)
-      .then(apis => this.createAppRegistration(args, apis, logger))
+      .then(apis => {
+        return this.createAppRegistration(args, apis, logger)
```

Comment:
We prefer not using nested promise chains, as they become harder to read. Could you refactor that?